main_routers/workshop_router.py

```
@@ -0,0 +1,1699 @@
+                logger.error(f"处理本地物品路径时出错: {e}")
+                return JSONResponse(content={"success": False, "error": f"路径处理错误: {str(e)}"}, status_code=500)
+        
+        return JSONResponse(content={"success": False, "error": "无效的物品ID格式"}, status_code=400)
+        
+    except Exception as e:
+        logger.error(f"获取本地创意工坊物品失败: {e}")
+        return JSONResponse(content={"success": False, "error": str(e)}, status_code=500)
+
+
+@router.get('/check-upload-status')
+async def check_upload_status(item_path: str = None):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)